static/scss/course-cards.scss

```diff
@@ -0,0 +1,105 @@
+// sass-lint:disable mixins-before-declarations
+.course-section h1 {
+  font-size: 44px;
+  font-weight: bold;
+  letter-spacing: 0;
+  line-height: 54px;
+  text-align: center;
+  padding-top: 54px;
+  padding-bottom: 54px;
+  margin: 0 auto;
+  max-width: 760px;
+
+  @include media-breakpoint-down(sm) {
+    font-size: 30px;
+    line-height: 40px;
+    padding-top: 20px;
+    padding-bottom: 22px;
+  }
+}
+
+.course-listing {
+  padding: 0;
+  margin: 0 -22px;
+  list-style: none;
+  letter-spacing: -0.32em;
+
+  .course-card-description {
+    overflow-y: scroll;
+    margin-bottom: 30px;
+  }
+
+  li {
+    @include media-breakpoint-down(md) {
+      width: calc(50% - 44px);
+    }
+
+    @include media-breakpoint-down(sm) {
+      width: calc(100% - 44px);
+    }
+    letter-spacing: normal;
+    display: inline-block;
+    vertical-align: top;
+    width: calc(33.33% - 44px);
+    margin: 0 22px 44px;
+
+    a {
```

Comment:
```suggestion
    .course-card {
```

Reply:
So we are now using `div` instead of `<a>`, let me know, is it not be clickable to redirect it to specific product ?